src/playwright-runner.js

```diff
@@ -263,6 +263,13 @@ async function run (nodeBin, runCfgPath, suiteName) {
     throw new Error(`Could not find projectPath directory: '${projectPath}'`);
   }
 
+  process.env.playwrightCfgFile = runCfg.playwright.configFile || '';
+  process.env.browserName = suite.param.browserName;
+
+  if (suite.param.project) {
+    process.env.project = suite.param.project;
```

Comment:
I still really think the logic around whether to use `--browser` and `--project` should be as close together as possible. The two options are mutually exclusive so our implementation should reflect that constraint as closely as possible.